src/utils/makeToken.ts

```diff
@@ -0,0 +1,8 @@
+import { sign } from "jsonwebtoken";
+import { JWT_SECRET } from "../config";
+
+function makeToken(payload: string | object | Buffer) {
```

Comment:
This function should be a part of the service for working with JWT. The refresh token can be added there, and also some other logic.

Reply:
the functionality has been moved to tokenService